pyiron_base/project/project_browser.py

```diff
@@ -0,0 +1,337 @@
+import os
+
+import ipywidgets as widgets
+from IPython.core.display import display
+
+from pyiron_base import Project as BaseProject
+from pyiron_base.generic.filedata import FileData
+
+
+__author__ = "Niklas Siemer"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "0.1"
+__maintainer__ = "Niklas Siemer"
+__email__ = "siemer@mpie.de"
+__status__ = "development"
+__date__ = "Feb 02, 2021"
+
+
+class ProjectBrowser:
+    """
+        Project Browser Widget
+
+        Allows to browse files/nodes/groups in the Project based file system.
+        Selected files may be received from this ProjectBrowser widget by the data attribute.
+    """
+    def __init__(self,
+                 project,
+                 Vbox=None,
+                 fix_path=False,
+                 show_files=True
+                 ):
+        """
+            ProjectBrowser to browse the project file system.
+
+            Args:
+                project: Any pyiron project to browse.
+                Vbox(:class:`ipython.widget.VBox`/None): Widget used to display the browser (Constructed if None).
+                fix_path (bool): If True the path in the file system cannot be changed.
+                show_files(bool): If True files (from project.list_files()) are displayed.
+        """
+        self.project = project.copy()
```

Comment:
I'm not yet super familiar with usage, so feel free to disregard this -- should `project` be write-protected as a private variable + `@property`?

Reply:
Then I actually do not need to copy, do I?

Reply:
Nope, should be safe